requirements.txt

```diff
@@ -1,2 +1,2 @@
-Requests==2.20.0
```

Comment:
i am not familiar with `httpx` but assuming you made this replacement b/c httpx has all functionalities `Requests` provides?

Reply:
`httpx` is heavily inspired by `requests` and attempts to have the same API where possible.  My motivation for using it is the fact that it has `Client` and `AsyncClient` which (mostly) share and API and produce the same response shape so that any code for parsing responses can be identical regardless of sync or async